jetson/nav/gimbal.cpp

```diff
@@ -0,0 +1,32 @@
+#include "gimbal.hpp"
+#include "iostream"
+using namespace std;
```

Comment:
in general its really bad practice to use this, get rid of` using namespace std` and import whatever you need from the stl in the form of `using std::vector` (for example), etc here instead or just prefix those functions/data types in the code with `std:: `